docker/test/stateless/stress_tests.lib

```diff
@@ -120,13 +120,41 @@ EOL
 
     local max_users_mem
     max_users_mem=$((total_mem*30/100)) # 30%
+    # Similar to docker/test/fuzzer/query-fuzzer-tweaks-users.xml
     echo "Setting max_memory_usage_for_user=$max_users_mem and max_memory_usage for queries to 10G"
-    cat > /etc/clickhouse-server/users.d/max_memory_usage_for_user.xml <<EOL
+    cat > /etc/clickhouse-server/users.d/stress_test_tweaks-users.xml <<EOL
 <clickhouse>
     <profiles>
         <default>
+            <max_execution_time>60</max_execution_time>
```

Comment:
We have this at 10 seconds for the fuzzer and had no limit in stress tests. I think limiting it to 60 seconds makes sense and would help reduce the number of "stuck queries" reports